CMakeLists.txt

```diff
@@ -30,6 +30,13 @@ project("Nebula Graph" C CXX)
 
 # Set the project home dir
 set(NEBULA_HOME ${CMAKE_CURRENT_SOURCE_DIR})
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
I think it would be better to detect if the host OS is one of our supported candidates, and print it out and exit if not.
Otherwise, _FreeBSD_ should be considered in this unsupported list.

Reply:
I am not sure about other Linux system, just disable for Darwin(MacOS), FreeBSD and Windows for now.

Reply:
Fair enough.